cmd/scaffold.go

```diff
@@ -502,6 +499,7 @@ func (fnb *FlowNodeBuilder) InitFlowNetworkWithConduitFactory(
 	}
 
 	fnb.Network = net
+	fnb.MisbehaviorReportConsumer = net
```

Comment:
```suggestion
	
```
Please remove `fnb.MisbehaviorReportConsumer` as it is unused.

Reply:
https://github.com/onflow/flow-go/pull/4512/commits/c5b2ff05d41722f1484fa367fa4495ee34afb4c6